tests/test_orient_symmetric.py

```diff
@@ -215,6 +253,30 @@ def testRelativeViewingDirections(L, order, dtype):
         (theta_vii, theta_vii_J, np.pi - theta_vii, np.pi - theta_vii_J), axis=0
     )
 
+    # For order>4 we cannot expect estimates from equator images to be accurate.
+    # So we exclude those from testing.
+    if order > 4:
+        equator_inds = np.array(
+            [
+                ind
+                for (ind, rot_gt) in enumerate(rots_gt)
+                if abs(np.arccos(rot_gt[2, 2]) - np.pi / 2) < 10 * np.pi / 180
+            ]
+        )
+
+        # Exclude ii estimates and ground truths.
+        min_theta_vii = np.delete(min_theta_vii, equator_inds, axis=0)
+        sii = np.delete(sii, equator_inds, axis=0)
+
+        # Exclude ij estimates and ground truths.
+        pairwise_equator_inds = []
+        for pair in pairs:
```

Comment:
Thanks! Just shaved 10 seconds off the short running testing time.

Reply:
Per our discussion, something like the following ( can use np.array(list()) instead of fromiter if you will want to use as array more, like below).
```
>>> from itertools import combinations
>>> n = 3
>>> np.fromiter(combinations(range(n),3), dtype='int,int,int')
array([(0, 1, 2)], dtype=[('f0', '<i8'), ('f1', '<i8'), ('f2', '<i8')])
>>> n = 10
>>> np.fromiter(combinations(range(n),3), dtype='int,int,int')
```

can prebake the look up tables
```
A = np.fromiter(combinations(range(n),3), dtype='int,int,int')
map_idx_to_triple = np.empty((len(A),3),dtype=int)
# could use dict, but array is more traditional (though wastes small amount of space)
map_triple_to_idx = np.full((len(A)-2,len(A)-1,len(A)), fill_value=-1, dtype=int)

for i,trip in enumerate(A):
    map_triple_to_idx[trip.tolist()] = i
    map_idx_to_triple[i] = trip.tolist()
```

You'd have to adjust the triples thing for whatever your i j k contraints are, this is probably backwards from what you need.

Reply:
Refactored `all_triplets` to use `itertools.combinations`. We're not using an index look up for triplets, so I left that part out. `combinations` was much faster than the list comprehension method. Thanks!